### DISCUSSION TITLE
UI issue, Manage Extended Attributes duplication / admin console view

### DISCUSSION THREAD

**Discussion by kuzono** [MasaCMS/MasaCMS#93] [Q&A]
I'm having an issue where Manage Extended Attributes is bugging out and duplicating itself multiple times across create (page, folder, component, ... etc). 
Does anyone know where the files are located for the admin side view? I would greatly appreciate some direction in finding the files so that I can see why this is happening. Thank you for your time. 

![image](https://user-images.githubusercontent.com/18051774/172165033-ee970f0d-723f-47f9-8d83-63b6f308694c.png)

![image](https://user-images.githubusercontent.com/18051774/172164949-6abc0d01-45d3-4f39-8817-74b08d571976.png)

**Comment by grantshepert** [comment]
I can't tell from the screenshot, but that looks like a mash-up of the old and new UI. Can you hard reload the page (i.e. clear your browser cache)? It might just be a CSS issue. If that doesn't fix it, is there an error in the console? That mark-up is correct, however the separate extension sections should hide/show depending upon which tab you are in.
